fugue-aws/src/main/java/com/symphony/oss/fugue/aws/sqs/SqsSubscriberManager.java

```diff
@@ -185,7 +185,7 @@ protected void initSubscription(ISubscription<String> subscription)
       String queueUrl = sqsClient_.getQueueUrl(subscriptionName.toString()).getQueueUrl();
       
       SqsSubscriber subscriber = new SqsSubscriber(this, sqsClient_, queueUrl, subscriptionName.toString(), getTraceFactory(), subscription.getConsumer(),
-          getCounter(), createBusyCounter(subscriptionName), nameFactory_.getPodName());
+          getCounter(), createBusyCounter(subscriptionName), null);
```

Comment:
This will break the former use case, and I don't think we need to make this change do we?
I think you are passing in a constructed config object and the podName will be null won't it?

Can you verify that podName is null and then revert this line please.